consensus/enclave/impl/src/lib.rs

```diff
@@ -600,6 +600,18 @@ impl ConsensusEnclave for SgxConsensusEnclave {
         // Try and deserialize.
         let tx: Tx = mc_util_serial::decode(&tx_bytes)?;
 
+        // Verify fee map digest if it is present.
+        if !tx.fee_map_digest.is_empty()
```

Comment:
:question:
Could someone strip out the fee map value between the client and here?  My understanding is the message is encrypted so it shouldn't happen and it would change the digest of the client message.

Reply:
Correct, the message is encrypted and has a MAC so it cannot be modified.